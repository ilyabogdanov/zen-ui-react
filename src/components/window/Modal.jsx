import React from "react";

/**
 * @class
 * @classdesc Modal
 * @author Ilya Bogdanov <public@ilyabogdanov.ru>
 * @license Apache-2.0
 *
 * @prop {object} props
 * @prop {*} props.children
 */
export default class Modal extends React.Component {
    render() {
        const { children } = this.props;
        return (
            <div className="zen_ui__modal">
                <div>
                    <div>
                        <div>{children}</div>
                    </div>
                </div>
            </div>
        );
    }
};