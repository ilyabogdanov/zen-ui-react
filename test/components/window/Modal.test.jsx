import React from "react";
import renderer from "react-test-renderer";
import Modal from "../../../src/components/window/Modal";

describe("Modal", () => {
    test("should render", () => {
        const e = renderer.create(
            <Modal/>
        ).toJSON();
        expect(e).toMatchSnapshot();
    });
});
