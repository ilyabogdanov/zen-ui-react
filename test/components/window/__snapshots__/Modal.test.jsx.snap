// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should render 1`] = `
<div
  className="zen_ui__modal"
>
  <div>
    <div>
      <div />
    </div>
  </div>
</div>
`;
